src/pages/office/office.html

```diff
@@ -1,9 +1,9 @@
 <ion-header>
   <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
   <ion-navbar hideBackButton>
-    <ion-title>Office</ion-title>
+    <ion-title>Test Summary</ion-title>
```

Comment:
This is a bit too high should be line 39 which is changed to Test summary from Test Summary